frontend/src/metabase/visualizations/echarts/cartesian/option/index.ts

```diff
@@ -35,6 +35,7 @@ export const getCartesianChartOption = (
   ];
 
   return {
+    animation: false,
```

Comment:
We can leave this for now, but we should have a TODO item in the future for consulting with product/design regarding what animations to keep and which ones to toss. Some animations like the transition between y-axis scales (linear, log, pow) I think are pretty neat in seeing how the data changes and are fairly unobtrusive.